spec/system/dashboard/callouts_spec.rb

```diff
@@ -0,0 +1,191 @@
+require 'rails_helper'
+
+RSpec.describe 'Callout pages', type: :system do
+  let(:user) { create(:user) }
+
+  before do
+    sign_in(user)
+  end
+
+  describe 'view all callout' do
+    it 'should list all callouts of current account' do
+      account_callout = create(:callout, account: user.account, metadata: { title: 'My Callout'})
+      other_callout = create(:callout, metadata: { title: 'Other Callout'})
+
+      visit dashboard_callouts_path
+
+      expect(page).to have_text('My Callout')
+      expect(page).not_to have_text('Other Callout')
+    end
+
+    it 'on click new callout will open new callout page' do
+      visit 'dashboard/callouts'
+
+      click_button 'New callout'
+      expect(page).to have_current_path(new_dashboard_callout_path)
+    end
```

Comment:
Better to write system/feature spec from user perspective and complete user stories.

In this case, just go to new callout page doesn't give any value, what they want from user perspective is be able to create callout.